lib/watch-files.js

```diff
@@ -24,16 +44,23 @@ module.exports = (nameOfDirectory) => {
     });
     watcher.on('change', async (file) => {
       let pattern;
+      const typeFile = file.replace(nameOfDirectory, '');
       if (nameOfDirectory === 'experience') {
-        const typeFile = file.replace(nameOfDirectory, '');
         const parsed = path.parse(typeFile);
         pattern = parsed.base;
         command.type = parsed.dir.replace(path.sep, '');
       } else {
-        pattern = file.replace(`${nameOfDirectory}${path.sep}`, `*${path.sep}`);
+        pattern = typeFile;
+      }
+
+      if (includes(file, queue)) {
+        queue.splice(queue.indexOf(file));
```

Comment:
I don't think you want to remove all the files in the queue after this one as well. Set the count to one.

Reply:
No we decided that if you have a file already in the queue and you try to queue the same file again, move it to the back of the queue. The queue should always be a unique set of files.

Reply:
I thought splice removed 1 element by default - didn't cause any obvious issues but I'll look at it

Reply:
Nope I see what your saying, yes that is true. This should be queue.splice(queue.indexOf(file), 1); My bad.